pkgs/development/compilers/mlton/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, fetchgit, patchelf, gmp }:
+{ stdenv, fetchurl, fetchgit, patchelf, gmp, which }:
 rec {
   mlton20130715 = import ./20130715.nix {
```

Comment:
shouldn't these actually be `callPackage`?

Reply:
Good point.  I've changed it.